### ISSUE TITLE
How to use the ARGMAX layer?

### ISSUE THREAD

**Issue by warmestwind** [BVLC/caffe#3939]
I use the net, as below, to predict alphabet. I want to know how can i get the classification result when i input an unlabeled image. Should i use the ARGMAX layer or what other methods?
------------------------------------------------------------------------------
name: "alpha"
layers {
  name: "alpha"
  type: IMAGE_DATA
  top: "data"
  top: "label"
  #image_data_param {
  data_param{ 
    source: "./models/ocr/training_images/LIST2.txt"
    batch_size: 500
    is_color: false
  }
  transform_param {
    scale: 0.00390625
  }
  include: { phase: TRAIN }
}
layers {
  name: "alpha"
  type: IMAGE_DATA
  top: "data"
  top: "label"
  image_data_param {
    source: "./models/ocr/training_images/LIST3.txt"
    batch_size: 500
    is_color: false
  }
  transform_param {
    scale: 0.00390625
  }
  include: { phase: TEST }
}

layers {
  name: "conv1"
  type: CONVOLUTION
  bottom: "data"
  top: "conv1"
  blobs_lr: 1
  blobs_lr: 2
  convolution_param {
    num_output: 20
    kernel_size: 5
    stride: 1
    weight_filler {
      type: "xavier"
    }
    bias_filler {
      type: "constant"
    }
  }
}
layers {
  name: "conv2"
  type: CONVOLUTION
  bottom: "conv1"
  top: "conv2"
  blobs_lr: 1
  blobs_lr: 2
  convolution_param {
    num_output: 50
    kernel_size: 5
    stride: 1
    weight_filler {
      type: "xavier"
    }
    bias_filler {
      type: "constant"
    }
  }
}
layers{
   name: "dropout"
   type: DROPOUT
   bottom: "conv2"
   top: "dropout"
} 
layers {
  name: "ip1"
  type: INNER_PRODUCT
  bottom: "dropout"
  top: "ip1"
  blobs_lr: 1
  blobs_lr: 2
  inner_product_param {
    num_output: 500
    weight_filler {
      type: "xavier"
    }
    bias_filler {
      type: "constant"
    }
  }
}
layers {
  name: "relu1"
  type: RELU
  bottom: "ip1"
  top: "ip1"
}
layers {
  name: "ip2"
  type: INNER_PRODUCT
  bottom: "ip1"
  top: "ip2"
  blobs_lr: 1
  blobs_lr: 2
  inner_product_param {
    num_output: 36
    weight_filler {
      type: "xavier"
    }
    bias_filler {
      type: "constant"
    }
  }
}
layers {
  name: "accuracy"
  type: ACCURACY
  bottom: "ip2"
  bottom: "label"
  top: "accuracy"
}
layers {
  name: "loss"
  type: SOFTMAX_LOSS
  bottom: "ip2"
  bottom: "label"
  top: "loss"
}

**Comment by seanbell** [comment]
From https://github.com/BVLC/caffe/blob/master/CONTRIBUTING.md:
> _Please do not post usage, installation, or modeling questions, or other requests for help to Issues._
Use the [caffe-users list](https://groups.google.com/forum/#!forum/caffe-users) instead. This helps developers maintain a clear, uncluttered, and efficient view of the state of Caffe.
